### ISSUE TITLE
Module not working at all

### ISSUE THREAD

**Issue by mimimi1968** [kiliankoe/dvbjs#42]
I'm setting up a MagicMirror2 and wanted to use the fine MMM-DVB module which is based on this library. No success at all :-(
I debugged with node.js (I'm a complete newbie to node...) and wireshark and finally found the solution: for some reason the 'proj4' module is picky when it comes to accept string parameters for conversion of coordinates. This is at least my opinion, see the attached patch which fixes the problem. After this patch all tests run by 'npm test' are OK. So HTH somebody else.
Thanks for your work on this software module!
```javascript
diff --git a/lib/utils.js b/lib/utils.js
index ea9e32d..0035c57 100644
--- a/lib/utils.js
+++ b/lib/utils.js
@@ -6,11 +6,11 @@ var proj4 = require('proj4');
 proj4.defs("GK4", "+proj=tmerc +lat_0=0 +lon_0=12 +k=1 +x_0=4500000 +y_0=0 +ellps=bessel +datum=potsdam +units=m");
 
 utils.WGS84toGK4 = function WGS84toGK4(lat, lng) {
-    return proj4("WGS84", "GK4", [lng, lat]).map(Math.round);
+    return proj4("WGS84", "GK4", [Number(lng), Number(lat)]).map(Math.round);
 };
 
 utils.GK4toWGS84 = function GK4toWGS84(lat, lng) {
-    return proj4("GK4", "WGS84", [lng, lat]).reverse();
+    return proj4("GK4", "WGS84", [Number(lng), Number(lat)]).reverse();
 };
 
 utils.stripSpaces = function stripSpaces(s) {
```

**Comment by kiliankoe** [comment]
Hi and thanks for the issue! Didn't even know about MMM-DVB, now I kinda want to build a MagicMirror as well!

Regarding the issue, that really shouldn't be happening 😕 I wonder why we haven't run into this before? 🤔 @ronhippler any ideas?

Would you mind opening a PR with the patch? Would love to pull this in if there's an actual issue preventing this library from working at all.

**Comment by kiliankoe** [comment]
I'm also seeing all tests passing on current master on my machine.

**Comment by kiliankoe** [comment]
btw: not sure if this has anything to do with the issue, but MMM-DVB relies on dvbjs@4.2.1 (which doesn't exist o.O). Current release is v4.3.5.

**Comment by mimimi1968** [comment]
Indeed strange...
I saw the old version dependency and my next step was to checkout master of dvbjs from github. This old version AFAIK had no tests so I was really full of hope when I started over with the latest version. But this didn't help, so I came into debugging. Btw if you can give some advice for tools or tricks and trades for effective debugging node.js projects, I would really appreciate this (a link of a good site is enough :-) ).
I will set up a fork/repo ASAP, so you can pull the patch. But a small concern remains for not understanding why this happens at all...

**Comment by ronhippler** [comment]
fixed with #43
